pyoptsparse/pyNLPQLP/pyNLPQLP.py

```diff
@@ -14,9 +14,6 @@
 from ..pyOpt_optimizer import Optimizer
 from ..pyOpt_error import Error
 
-# constants
-eps = np.finfo(np.float64).eps
```

Comment:
what is this?

Reply:
This is [machine epsilon](https://en.wikipedia.org/wiki/Machine_epsilon). We use it to compare design variables during hot start, for example.